agent/src/main/java/nu/studer/teamcity/buildscan/agent/BuildScanServiceMessageInjector.java

```diff
@@ -275,7 +288,7 @@ private MavenExtensions getMavenExtensions(BuildRunnerContext runner) {
             return MavenExtensions.empty();
         }
 
-        final List<File> searchLocations = new ArrayList<File>();
+        final List<File> searchLocations = new ArrayList<>();
```

Comment:
Java 6 / Diamond Operator

```suggestion
        final List<File> searchLocations = new ArrayList<File>();
```

Reply:
Changed in https://github.com/etiennestuder/teamcity-build-scan-plugin/pull/373/commits/90f96034f60ab2301893309222081f61a13f25fc